sky/skypilot_config.py

```diff
@@ -154,27 +246,76 @@ def _reload_config() -> None:
                     'exist. Please double check the path or unset the env var: '
                     f'unset {ENV_VAR_SKYPILOT_CONFIG}')
     else:
-        config_path = CONFIG_PATH
```

Comment:
can we just raise an exception actually, or assertion? we only run this function if we have the env var

Reply:
actually it's cleaner to just have the path be a function argument